lib/beaker/host/unix/pkg.rb

```diff
@@ -27,7 +27,7 @@ def check_for_package(name, opts = {})
     case self['platform']
       when /sles-10/
         result = execute("zypper se -i --match-exact #{name}", opts) { |result| result }
-        /No packages found/.match?(result.stdout) ? (return false) : (return result.exit_code == 0)
+        result.stdout.include?('No packages found') ? (return false) : (return result.exit_code == 0)
```

Comment:
Perhaps not for this PR, but this can be simplified to:
```suggestion
        return !result.stdout.include?('No packages found') && result.exit_code == 0
```